NiksVeinbergs/m2
# Ml 1413 Pre-Footer Text to CMS Page

Add Pre-Footer Text to CMS Page #1

Description
The new function can be accessed through the Content->Pages->Edit Page->Pre-Footer Text fieldset. There should be an editable WYSIWYG form element.
Pre-Footer Text is saved in the "cms_page" table new field "pre_footer_text" (Added via SchemaPatch)
If Pre-Footer Text is added, it should be visible at the end of the CMS Page. (Before footer)

Manual testing scenarios
Create new Pre-Footer Element in Content->Pages->Edit Page->Pre-Footer Text fieldset
Go to the page and check if the Pre-Footer block is visible at the end of the page.
Delete text in the Pre-Footer text fieldset
Check if text disappears on the CMS page
